src/components/Editor/SearchBar.js

```diff
@@ -122,8 +121,7 @@ class SearchBar extends Component {
       shiftSearchAgainShortcut
     } = getShortcuts();
 
-    shortcuts.on(searchShortcut, (_, e) => this.toggleSearch(e));
```

Comment:
Did you mean to remove the search shortcut?

Reply:
yes, because the search bar is now unmounted when it is not open. See the Editor/index render method

Reply:
I see that. But now there isn't a way to enable the search bar?

Reply:
oh, we should probably fix that :) 

I think i was going to add a shortcut in App.js or Editor/index